.github/workflows/main.yml

```diff
@@ -0,0 +1,141 @@
+name: Continous Integration Tests
+
+on:
+  push:
+    branches:
+      - master 
+      - release
+  pull_request:
+    branches:
+      - master 
+      - release
+jobs:
+  lint:
+    strategy:
+      fail-fast: false #ensure both tests run even if one fails
+      matrix:
+        os: [ubuntu-latest, windows-latest]   
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Cache NPM packages 
```

Comment:
No use doing this because `npm ci` doesn't care about caches

Reply:
Yes